src/operator/cudnn_convolution.cc

```diff
@@ -11,6 +11,7 @@
 namespace mxnet {
 namespace op {
 #if MXNET_USE_CUDNN == 1
+// TODO(xxx): Refactor with Init CuDNN function, remove redandent code in initalization
```

Comment:
I did this intentionally so that it would be easier to isolate this into it's own tune class